pkg/util/ssh/ssh.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var (
-	verbose = flag.Bool("ssh-verbose", false, "show the verbose of SSH command")
+	verbose bool
```

Comment:
Why?

Reply:
because it will register a flag to every binary, you can see it from --help. Most of our cases run on K8s, so `ssh-verbose` is redundant now.

Reply:
```
func init() {
	flag.BoolVar(&verbose, "ssh-verbose", false, "show the verbose of SSH command")
}
```

When package is import, init will be executed.

Reply:
Yes, so only on need.